background.js

```diff
@@ -17,4 +17,24 @@ chrome.runtime.onMessage.addListener(function(message, sender, callback) {
     callback(null, err)
   })
   return true; // need for async callback
+})
+
+const youtubeApiUrl = 'https://raw.githubusercontent.com/FeignedAccomplice/YOUTUBOTS/master/KB.CSV';
+
+chrome.runtime.onMessage.addListener(function(message, sender, callback) {
```

Comment:
А что, у нас нет ES6? Предалагаю:

```suggestion
chrome.runtime.onMessage.addListener((message, sender, callback) => {
```